it/connectors/propertyLinking/ManageAgentISpec.scala

```diff
@@ -0,0 +1,335 @@
+package connectors.propertyLinking
```

Comment:
This is a controller test, not a connector one, isn't it?

Reply:
Yes.  I'll  place it (and the 2 others) in the controller package